src/lib.rs

```diff
@@ -1350,3 +1391,8 @@ impl Iterator for Varbinds {
         None
     }
 }
+
+struct ResponseItemInt {
```

Comment:
Suffix `Int` is misleading, assumes there is an integer in response but it's a vec. You can call it `ResponsePacket` perhaps or similar.

Reply:
ok